target/linux/mvebu/image/cortexa9.mk

```diff
@@ -232,3 +232,18 @@ define Device/solidrun_clearfog-pro-a1
   BOOT_SCRIPT := clearfog
 endef
 TARGET_DEVICES += solidrun_clearfog-pro-a1
+
+define Device/kobol_helios4
+  DEVICE_VENDOR := Kobol
+  DEVICE_MODEL := Helios4
+  KERNEL_INSTALL := 1
+  KERNEL := kernel-bin
+  DEVICE_PACKAGES := mkf2fs e2fsprogs partx-utils swconfig
+  IMAGES := sdcard.img.gz
+  IMAGE/sdcard.img.gz := boot-scr | boot-img-ext4 | sdcard-img-ext4 | gzip | append-metadata
+  DEVICE_DTS := armada-388-helios4
```

Comment:
Should work with `SOC := armada-388` instead.

Reply:
uhm, not sure what you mean. This device has a specific dts, and I think I need to specify its name here.

Reply:
image/Makefile contains `DEVICE_DTS = $$(SOC)-$(lastword $(subst _, ,$(1)))`, so the DTS file name will be constructed from the device name and the SOC ...

Reply:
ok